darwin/client.py

```diff
@@ -1080,7 +1080,7 @@ def _raise_if_known_error(self, response: Response, endpoint: str) -> None:
                 raise InsufficientStorage()
 
     def _has_json_response(self, response: Response) -> bool:
-        return response.headers.get("content-type") == "application/json"
+        return "application/json" in str(response.headers.get("content-type"))
```

Comment:
I am converting to `str` because `response.headers.get("content-type")` can return `None`. In such an event, this raise. 
However, if I convert `str(None)` I get `"None"`, in which case the condition will return `False` which is what I want.

Reply:
what about 
```suggestion
        return "application/json" in response.headers.get("content-type", "")
```

but hey no strong opinion, still consider it approved